src/Microsoft.AspNet.Razor.Runtime/TagHelpers/TagHelperExecutionContext.cs

```diff
@@ -22,12 +22,13 @@ public class TagHelperExecutionContext
         /// <summary>
         /// Internal for testing purposes only.
         /// </summary>
-        internal TagHelperExecutionContext(string tagName)
+        internal TagHelperExecutionContext(string tagName, bool selfClosing)
             : this(tagName,
```

Comment:
I'm still not sure if I should keep this signature like TagHelperExecutionContext(string tagName), and initialize selfClosing with its the default value (false) just like any other parameters. For now, I changed its signature to simplify some test codes. But, let me know.